source/publishing_data/12_translating_dataset/translating_dataset.rst

```diff
@@ -0,0 +1,98 @@
+Translating a dataset
+=====================
+
+For domains with more than one language activated in the front office, Opendatasoft allows the translation of specific elements of the datasets, as well as their metadata. When translated, these information appear in the language the user selected to navigate the portal. It should help them understand the dataset as best as possible, even though the data itself is not in their preferred language.
+
+In the Opendatasoft platform, these translations are done in the Translations tab of the edition interface of each dataset.
+
+.. image:: images/translation_tab_interface.png
+
+Choosing elements for translation
+---------------------------------
+
+For each available language displayed in the Translations tab, it is possible to translate various elements that have been originally configured in 2 other configuration tabs of the dataset.
+
+- The :doc:`Information tab</publishing_data/06_configuring_metadata/configuring_metadata>`, which contains 4 standard metadata that can be translated: Title, Description, Keywords and Publisher.
+- The :doc:`Processing tab</publishing_data/05_processing_data/processing_data>`, which contains the dataset fields names and descriptions, that can also be translated.
```

Comment:
Usually in the platform, the field names are the identifiers (e.g. `tree_name`), the displayed name is called the label, so we can say "the dataset fields labels and descriptions"